docs/tasks/stateful-sets/debugging-a-statefulset.md

```diff
@@ -0,0 +1,78 @@
+---
+assignees:
+- bprashanth
+- enisoc
+- erictune
+- foxish
+- janetkuo
+- kow3ns
+- smarterclayton
+
+---
+
+{% capture overview %}
+This task shows you how to debug a Stateful Set.
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* If you are not using Stateful Sets, you can skip this task now. 
+
+{% endcapture %}
+
+{% capture steps %}
+
+### Debugging a Stateful Set
+
+In order to list all the pods which belong to a Stateful Set, which have a label `app=myapp` set on them, you can use the following: 
+
+```shell
+kubectl get pods -l app=myapp
+```
+If you find that any Pods listed are in `Unknown` or `Terminating` state for an extended period of time, refer to the [Deleting Stateful Set Pods](/docs/tasks/stateful-sets/deleting-pods/) task for instructions on how to deal with them. You can debug individual Pods in a Stateful Set using the [debugging Pods guide](docs/user-guide/debugging-pods-and-replication-controllers/#debugging-pods). 
```

Comment:
The two links to other pages are inconsistent in capitalization and inclusion of "task" or "guide" in the linkified span.